Makefile

```diff
@@ -345,4 +345,4 @@ check-doc-links:
 	@python3 hack/verify-doc-links.py && echo "$@: OK"
 
 poetry-update-lockfiles:
-	bash -ec 'for value in $$(find . -name poetry.lock -exec dirname {} \;); do (cd "$${value}" && echo "Updating $${value}/poetry.lock" && poetry update --lock); done'
+	bash -ec 'for value in $$(find . -name poetry.lock -exec dirname {} \;); do (cd "$${value}" && echo "Updating $${value}/poetry.lock" && poetry lock --no-update); done'
```

Comment:
Could you clarify why this change is needed?

Reply:
it is not :)
reverted back